src/pgduckdb_types.cpp

```diff
@@ -30,94 +30,72 @@ extern "C" {
 
```

Comment:
The file mode was changed to be executable (i.e. from 644 to 755). This seems by accident. Can you undo that?

Same issue for `array_type_support.sql`.